test/integration/include/findAll.test.js

```diff
@@ -1172,7 +1173,8 @@ describe(Support.getTestDialectTeaser('Include'), () => {
       expect(products[0].Tags.length).to.equal(1);
     });
 
-    it('should be possible to extend the on clause with a where option on nested includes', async function () {
+    // Assertion Error for yugabyte due to the expection of rows to be selected in the same way it is inserted in the database can't be possible
```

Comment:
nit: you can explain it at few places and just use "Assertion Error for yugabytedb" for other places